From 85b76426fd84b27e0343f5e6c43584d76bb01619 Mon Sep 17 00:00:00 2001
From: David Stansby <dstansby@gmail.com>
Date: Thu, 22 Jun 2023 09:39:17 +0100
Subject: [PATCH] Docs for adding LayerData tuple to viewer (#152)

# Description
This adds some short docs for adding a LayerData tuple to the napari
viewer. Replaces https://github.com/napari/napari/pull/4711.
---
 docs/plugins/_layer_data_guide.md | 13 ++++++++++++-
 1 file changed, 12 insertions(+), 1 deletion(-)

diff --git a/docs/plugins/_layer_data_guide.md b/docs/plugins/_layer_data_guide.md
index 000a0ba6..12fb656d 100644
--- a/docs/plugins/_layer_data_guide.md
+++ b/docs/plugins/_layer_data_guide.md
@@ -117,4 +117,15 @@ Out[7]:
     },
     'image'
 )
-```
\ No newline at end of file
+```
+
+### Adding to the viewer
+
+To add a `LayerData` tuple to the napari viewer, use :meth:`Layer.create`:
+
+```python
+>>> image_layer_data = (data, {'name': 'My Image', 'colormap': 'red'}, 'image')
+>>> viewer = napari.current_viewer()
+>>> viewer.add_layer(napari.layers.Layer.create(*image_layer_data))
+```
+The only attribute that can't be passed to `napari.layers.Layer.create` that is otherwise valid for a `LayerData` tuple is 'channel_axis'.
